github-actions/verify-pr-title/test/verifyFeaturePrTitle.test.ts

```diff
@@ -103,42 +102,42 @@ describe('#verifyFeaturePrTitle', () => {
 	/**
 	 * The Scope portion is the second part of the title.
 	 * E.g., in "NEW (PMD) @W-1234@ - Added whatever", the Scope portion is "(PMD)".
-	 * The Scope can be anything the author wants, but our general hope is that it would correspond
-	 * to a product domain (e.g., PMD, ESLint, etc).
+	 * The Scope can be anything the author wants, and is technically optional, but our general hope is that it would
+	 * correspond to a product domain (e.g., PMD, ESLint, etc).
 	 */
 	describe('Scope portion', () => {
 		function createTitle(scope: string): string {
 			return `NEW ${scope} @W-1234@ Filler text`;
 		}
 
-		it('Scope portion cannot be absent', () => {
+		it('Scope portion may absent', () => {
 			const title = createTitle('');
-			expect(verifyFeaturePrTitle(title)).to.equal(false, 'Should be rejected');
+			expect(verifyFeaturePrTitle(title)).toEqual(true);
```

Comment:
This is the only test that changed non-cosmetically. All other tests in all other files were simply converted from `chai`'s `expect` implementation to `jest`'s.